hakimel/reveal.js
# fix search plugin ignoring diacritics

allows searching for diacritics as asked in #3492

to test it : 
1. `npm start` 
2. add a slide in the `demo.html` file : 
    ```html
    <section>
        <h2>DIACRITICS SEARCH</h2>
        <p style="word-wrap: break-word">
            ÁĂẮẶẰẲẴǍÂẤẬẦẨẪÄẠÀẢĀĄÅǺÃÆǼBḄƁʚɞCĆČÇĈĊƆʗDĎḒḌƊḎǲǅĐÐǱǄEÉĔĚÊẾỆỀỂỄËĖẸÈẺĒĘẼƐƏFƑ
            GǴĞǦĢĜĠḠʛHḪĤḤĦIÍĬǏÎÏİỊÌỈĪĮĨĲJĴKĶḲƘḴLĹȽĽĻḼḶḸḺĿǈŁǇMḾṀṂNŃŇŅṊṄṆǸƝṈǋÑǊOÓŎǑÔỐỘỒỔỖÖỌ
            ŐÒỎƠỚỢỜỞỠŌƟǪØǾÕŒɶPÞQRŔŘŖṘṚṜṞʁSŚŠŞŜȘṠṢẞTŤŢṰȚṬṮŦÞÐUÚŬǓÛÜǗǙǛǕỤŰÙỦƯỨỰỪỬỮŪŲ
            ŮŨVWẂŴẄẀʬXYÝŶŸẎỴỲƳỶȲỸZŹŽŻẒẔƵlaáăắặằẳẵǎâấậầẩẫäạàảāąåǻãæǽɑɐɒbḅɓßcćčçĉɕċdďḓḍɗḏđɖʤǳʣ
            ʥǆðeéĕěêếệềểễëėẹèẻēęẽʒǯʓɘɜɝəɚʚɞfƒſʩﬁﬂʃʆʅɟʄgǵğǧģĝġɠḡɡɣhḫĥḥɦẖħɧɥʮʯųiíĭǐîïịìỉīįɨĩɩıĳɟjǰĵʝȷɟʄkķḳƙḵĸʞlĺƚɬ
            ľļḽḷḹḻŀɫɭłƛɮǉʪʫmḿṁṃɱɯɰnŉńňņṋṅṇǹɲṉɳñǌŋŊoóŏǒôốộồổỗöọőòỏơớợờởỡōǫøǿõɛɔɵʘœpɸþqʠrŕřŗṙṛṝɾṟɼɽɿɹɻɺ
            sśšşŝșṡṣʂſʃʆßʅtťţṱțẗṭṯʈŧʨʧþðʦʇuʉúŭǔûüǘǚǜǖụűùủưứựừửữūųůũʊvʋʌwẃŵẅẁʍxyýŷÿẏỵỳƴỷȳỹʎzźžʑżẓẕ
            ƶIǂʭʔʡʖǁʕǃʢ
        </p>
    </section>
    ```
3. load presentation in browser http://localhost:8000/demo.html 
4. using search plugin, search for any diacritic, it should navigate to this slide and highlight the right text

Please note that it may not work for every diactritics in every character set (only tested on latin diacritics). 
Anyway maybe the discussion opened at hakimel/reveal.js#2331 can make the problem much simpler if we decide to search any character without altering the user input string.

fixes hakimel/reveal.js#3492
